chef/cookbooks/keystone/libraries/helpers.rb

```diff
@@ -3,8 +3,11 @@ def self.service_URL(node, host, port)
     "#{node[:keystone][:api][:protocol]}://#{host}:#{port}"
   end
 
-  def self.versioned_service_URL(node, host, port)
-    service_URL(node, host, port) + '/' + node[:keystone][:api][:version] + '/'
+  def self.versioned_service_URL(node, host, port, version=nil)
```

Comment:
Can't we set default value directly here, as ``version=node[:keystone][:api][:version]`` ?